toolkit/src/components/cut/filter-bar/filter/filter-input.hbs

```diff
@@ -0,0 +1,25 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+}}
+
+{{#let @components as |Components|}}
+  <Components.Header @hasDivider={{true}}>
+    <Hds::Form::TextInput::Base
+      @type='search'
+      @value={{@value}}
+      style='margin: 8px 0;'
+      placeholder='Search'
+      aria-label='Search Clusters'
+      {{on 'input' @onInput}}
+    />
+  </Components.Header>
+  <Components.Generic>
+    <span
+      class='hds-foreground-faint hds-typography-body-100 hds-font-weight-regular'
+      style='display: flex; justify-content: space-between; margin: 8px 0;'
+    >
+      <span>Cluster ID</span>
```

Comment:
Cluster ID might be probably added as the param, in case we will use the filter for some other entity